docs/report-entries.adoc

```diff
@@ -110,4 +110,31 @@ The extension will publish your report entry if the test fails.
 ON_ABORTED::
 The extension will publish your report entry if the test was aborted (for example because of an https://junit.org/junit5/docs/current/user-guide/#writing-tests-assumptions[Assumption]).
 
-Entries will always be published after the test finished.
\ No newline at end of file
+Entries will always be published after the test finished.
+
+=== Publishing test parameters
+
+You can access `@ParameterizedTest` parameter values and publish them with the `@ReportEntry` annotation.
+Add a number (starting from zero) between curly braces to reference the parameter with that index.
```

Comment:
I would add an hint like "Inside the report entry the parameter values can be reported in any order".

Question from my side: Can they be reported multiple times, e.g. `{1} - {1} - {0} - {0}`, ?

Reply:
Yes, they can be reported multiple times, I'll update the tests (and the documentation) to reflect that.